Kernel/Modules.cmake

```
@@ -0,0 +1,162 @@
+if (NOT EXISTS "${CMAKE_SOURCE_DIR}/Build/${SERENITY_ARCH}/Kernel/config.ini")
+    configure_file("${CMAKE_SOURCE_DIR}/Kernel/config.ini.template" "${CMAKE_SOURCE_DIR}/Build/${SERENITY_ARCH}/Kernel/config.ini" COPYONLY)
+endif()
+set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS "${CMAKE_SOURCE_DIR}/Build/${SERENITY_ARCH}/Kernel/config.ini")
```

This is a bit awkward with the hard-coded binary directory. namely, it won't work when you have a Clang build. 

```suggestion
if (NOT EXISTS "${CMAKE_CURRENT_BINARY_DIR}/config.ini")
    configure_file("${CMAKE_SOURCE_DIR}/Kernel/config.ini.template" "${CMAKE_CURRENT_BINARY_DIR}/config.ini" COPYONLY)
endif()
set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/config.ini")
```